tests/storage/upgrade/test_upgrade_storage.py

```diff
@@ -61,21 +67,26 @@ def test_cdiconfig_scratch_overriden_before_upgrade(
     def test_vm_snapshot_restore_before_upgrade(
         self,
         skip_if_no_storage_class_for_snapshot,
-        cirros_vm_for_upgrade_a,
+        rhel_vm_for_upgrade_a,
         snapshots_for_upgrade_a,
     ):
         with VirtualMachineRestore(
-            name=f"restore-snapshot-{cirros_vm_for_upgrade_a.name}",
+            name=f"restore-snapshot-{rhel_vm_for_upgrade_a.name}",
             namespace=snapshots_for_upgrade_a.namespace,
-            vm_name=cirros_vm_for_upgrade_a.name,
+            vm_name=rhel_vm_for_upgrade_a.name,
             snapshot_name=snapshots_for_upgrade_a.name,
         ) as vm_restore:
             vm_restore.wait_restore_done()
-            cirros_vm_for_upgrade_a.start(wait=True)
-            run_command_on_cirros_vm_and_check_output(
-                vm=cirros_vm_for_upgrade_a,
-                command=LS_COMMAND,
-                expected_result="1",
+            rhel_vm_for_upgrade_a.start(wait=True)
+            write_file_via_ssh(
+                vm=rhel_vm_for_upgrade_a,
+                filename=UPGRADE_STORAGE_TEST_FILE_NAME_A,
+                content=UPGRADE_STORAGE_TEST_FILE_CONTENT,
+            )
```

Comment:
The idea is to write something to the VM, create a snapshot, restore VM from the snapshot, and make sure what we wrote is still there. So there's no need to write anything after we restored the snapshot.